### PR TITLE
updated dex image tag to add ppc64le support

### PR DESCRIPTION
**Pull request by mayurwaghmode**
Closes https://github.com/argoproj/argo-cd/pull/10617

### CODE REVIEW

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]

**Comment by 34fathombelow** [comment]
@crenshaw-dev please see https://github.com/dexidp/dex/pull/2352 &  https://github.com/dexidp/dex/issues/2613.  Dexv2.33 introduced some changes to gitlab tokens, can we verify this will not effect our users.

File: manifests/base/dex/argocd-dex-server-deployment.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ spec:
             type: RuntimeDefault
       containers:
       - name: dex
-        image: ghcr.io/dexidp/dex:v2.32.0-distroless
+        image: ghcr.io/dexidp/dex:v2.34.0
```
[the remark this answers is not in the document]
Comment (mayurwaghmode) [reply manifests/base/dex/argocd-dex-server-deployment.yaml:40]: yes, it is there. Let me modify it :)

**Comment by crenshaw-dev** [comment]
@34fathombelow ah yep I think you'd mentioned this before, and I forgot. Taking a look...

File: manifests/base/dex/argocd-dex-server-deployment.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mayurwaghmode) [reply manifests/base/dex/argocd-dex-server-deployment.yaml]: @crenshaw-dev I have updated it with distroless image tag

**Comment by crenshaw-dev** [comment]
I'm hesitant to merge until I understand the Dex bug better or until a fix is merged. Not sure if I'll have time to investigate before 2.5.

The good news is, it's fairly easy for folks to override the tag for that image and run the newer Dex version if they're not worried about the bug @34fathombelow linked.

**Comment by 34fathombelow** [comment]
@mayurwaghmode Argo CD 2.4.13 now uses dex:v2.35.0 which will support ppc64le. :)

**Comment by mayurwaghmode** [comment]
> @mayurwaghmode Argo CD 2.4.13 now uses dex:v2.35.0 which will support ppc64le. :)

Thank you @34fathombelow 💯

**Pull request closed without merging**